README.md

```diff
@@ -130,6 +130,7 @@ Each rule has emojis denoting:
 | [no-inline-styles](./docs/rule/no-inline-styles.md)                                                       | ✅  |     |     |     |
 | [no-input-block](./docs/rule/no-input-block.md)                                                           | ✅  |     |     |     |
 | [no-input-tagname](./docs/rule/no-input-tagname.md)                                                       | ✅  |     |     |     |
+| [no-invalid-aria-attributes](./docs/rule/no-invalid-aria-attributes.md)                                                     |     |     | ⌨️  |     |
```

Comment:
The whitespace formatting is off for this new line, which is causing rendering issues for all the subsequent table rows.

Reply:
`yarn update:readme` to fix this.